templates/endpoints_v2/examples/CustomModels.ipynb

```diff
@@ -51,7 +51,7 @@
     "\n",
     "The string template should include the `{instruction}` keyword, which will be replaced with message content from the ChatCompletions API.\n",
     "\n",
-    "For example, if a user sends the following message for llama2-7b-chat-hf ([prompt format](models/llama/meta-llama--Llama-2-7b-chat-hf_a10g_tp1.yaml#L30-L36)):\n",
+    "For example, if a user sends the following message for `llama2-7b-chat-hf` ([prompt format](models/llama/meta-llama--Llama-2-7b-chat-hf_a10g_tp1.yaml#L30-L36)):\n",
```

Comment:
oh this link may be wrong now. Can you check?

Reply:
You're right. I update this section.